templates/nextflow-ecs-task-definition.yaml

```diff
@@ -46,29 +43,89 @@ Parameters:
     Type: String
     Description: MySQL DB password
     NoEcho: 'true'
+  TowerGithubClientId:
+    Type: String
+    Description: The client id provided by GitHub for the OAuth app
+    NoEcho: 'true'
+  TowerGithubSecret:
+    Type: String
+    Description: The client secret provided by GitHub for the OAuth app
+    NoEcho: 'true'
+  RedisContainerName:
+    Type: String
+    Description: (Optional) Name of the container that runs redis
+    Default: redis
+  RedisContainerImage:
+    Type: String
+    Description: (Optional) Redis container docker image
+    Default: 'redis:5.0.8'
```

Comment:
seems better to make these container image strings required params and explicitly pass in the image names because you will probably want to eventually update and make them different for each environment.